apis/accounts.es6.js

```diff
@@ -1,27 +1,32 @@
-import BaseEndpoint from '../apiBase/BaseEndpoint';
-import Account from '../models/account';
+import { runQuery } from '../apiBase/APIRequestUtils';
+import Account from '../models2/account';
 
-export default class AccountsEndpoint extends BaseEndpoint {
-  move = this.notImplemented('move');
-  copy = this.notImplemented('copy');
-  put = this.notImplemented('put');
-  patch = this.notImplemented('patch');
-  post = this.notImplemented('post');
-  del = this.notImplemented('del');
-
-  path(method, query={}) {
-    if (query.user === 'me') {
-      return 'api/v1/me';
-    } else {
-      return `user/${query.user}/about.json`;
-    }
+const getPath = (query) => {
+  if (query.loggedOut) {
+    return 'api/me.json';
+  } else if (query.user === 'me') {
+    return 'api/v1/me';
+  } else {
```

Comment:
technically `no-else-return` isn't in the `.eslintrc` in this repo. I don't have the reddit-mobile linter globally so there's some linter errors slipping by, sorry :/ After I clean up this patch I'll make a separate pr to pull in the latest linter.